decide/administration/serializers.py

```diff
@@ -29,10 +32,11 @@ class UserUpdateSerializer(serializers.Serializer):
 
 class AdminQuestionSerializer(serializers.ModelSerializer):
     options = AdminQuestionOptionSerializer(many=True)
+
     class Meta:
         model = Question
-        fields = ('id','desc', 'options')
-        depht=1
+        fields = ('id', 'desc', 'options')
+        depht = 1
```

Comment:
this is misspelt, I guess you mean **_depth_**

Reply:
Typo fixed in last commit

Reply:
still being _depht_ instead of **_depth_** @pedalopon